Policies/github-terms-of-service.md

```diff
@@ -46,6 +46,7 @@ Effective date:
 4. “The User,” “You,” and “Your” refer to the individual person, company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the account; or that directs the use of the account in the performance of its functions. A User must be at least 13 years of age. Special terms may apply for business or government accounts (See [Section B(4): Additional Terms](#4-additional-terms)).
 5. “GitHub,” “We,” and “Us” refer to GitHub, Inc., as well as our affiliates, directors, subsidiaries, contractors, licensors, officers, agents, and employees.
 6. “Content” refers to content featured or displayed through the Website, including without limitation text, data, articles, images, photographs, graphics, software, applications, designs, features, and other materials that are available on the Website or otherwise available through the Service. "Content" also includes Services. “User-Generated Content” is Content, written or otherwise, created or uploaded by our Users. "Your Content" is Content that you create or own. “Paid Content” is Content only available to Users who are participating in a payment plan, including private repositories.
+7. “Free Licence” refers to a Free / Open Source / Open Knowledge / Ⓕ Copyfree licence. For practicality reasons, the set of Free Licences is comprised of the licences listed at [ifrOSS](http://www.ifross.org/en/license-center), [Debian](https://wiki.debian.org/DFSGLicenses), [OKFN](http://opendefinition.org/licenses/), [OSI](https://opensource.org/licenses/alphabetical), [FSF](https://www.gnu.org/licenses/license-list.html), [the Copyfree Initiative](http://copyfree.org/standard/licenses), and/or the [list of licences approved for Free Cultural Works](http://freedomdefined.org/Licenses), explicitly excluding those listed on these sites as not Free (such as those under the headings “B. Other Software Licenses”, “D. Other Licenses for Fair Use of Immaterial Goods”, “DFSG-incompatible”, “Non-Conformant Licenses”, “Nonfree Software Licenses”, “Nonfree Documentation Licenses”, “Licenses for Works stating a Viewpoint”, “Invariant sections” in the linked lists).
```

Comment:
> The list of lists is not perfect. However, it is a start, it doesn’t contain any false positives, it doesn’t need GitHub to finally disclose what rights their lawyers think they precisely need, and it fixes the problem the new ToS introduced in February for most OSS projects.

Anything that gives us a list of already-sufficiently-free licenses would be very useful.  But @nsqe [doesn't want a local list of licenses][1], and delegation like you're proposing here seems dicey in the absence of a clear definition of the rights GitHub needs granted.

> Also, note that forks do not get “free redistribution”, only inside GitHub, by the default licence.

Right.  But are any of the license-classification groups you link making that distinction?

> I think we can believe that anything that passes one set of those criteria and does not fail others’ sets in those list is “good enough” to be promoted to first-class citizen and thus explicitly excluded from the extra licence clause.

As far as users are concerned, I'd rather look at the project license directly instead of going through the ToS (assuming those are even binding).  So it doesn't matter as much to me.

GitHub, on the other hand, is doing a lot of automatic stuff based on the ToS, and they might be more cautious about delegating authority to so many external organizations.

[1]: https://github.com/github/site-policy/issues/7#issuecomment-313810026